notebooks/byod.py

```diff
@@ -0,0 +1,224 @@
+# publish to: "terra-notebook-utils-tests" "test-byod"
+import os
+import callysto
+
+# Mock the notebook environment
+os.environ['WORKSPACE_NAME'] = "terra-notebook-utils-tests"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-9169fcd1-92ce-4d60-9d2d-d19fd326ff10"
+os.environ['GOOGLE_PROJECT'] = "firecloud-cgl"
+
+with callysto.Cell("markdown"):
+    """
+    # Bring your own data to your Terra workspace and organize it in a data table
+    """
+
+with callysto.Cell("markdown"):
+    """
+    If you are planning to upload many files to your Terra workspace, we recommend you organize your data into a Terra
+    data table. This is especially helpful when you plan to run workflows with this data because you can avoid pasting
+    in the "gs://" links to every file and instead use Terra's helpful UI features.
+
+
+    In this example, we introduce tools to help you:
+
+    1. Programmatically upload data from your local machine to your Terra workspace using `gsutil cp`.
```

Comment:
We definitely duplicated work in the creation of the tables, but this part and mine differ in how the files get into the bucket. If we combine efforts I think this part will go unchanged; it's much better than my sort of handwaved "just get them in the bucket somehow" approach.